tabular/lending/configs/incremental_config

```diff
@@ -0,0 +1,4 @@
+{
+    "eval_path": rime_testing_path,
```

Comment:
are we checking in this file? lots of issues with it atm, maybe we shouldn't for now

Reply:
I don't need either configs for the notebook to work so I'm happy to leave this out